### ISSUE TITLE
Kite won't launch after fresh install (despite no errors in console)

### ISSUE THREAD

**Issue by davide-zagami** [kiteco/issue-tracker#206]
Hi Kite folks,

I'm running a 64bit Debian 10.3 with XFCE 4.12, and Miniconda 4.8.3 with Python 3.7.

I installed Kite and I got this:

![kite_install](https://user-images.githubusercontent.com/42622067/81279603-a6032580-9057-11ea-98b7-865bbbf6e090.png)

So all good, it seems. But Kite's window won't appear. A look at active processes shows this:

![kite_procs_after_install](https://user-images.githubusercontent.com/42622067/81279637-b0bdba80-9057-11ea-88d2-c6bfcea6f4ba.png)

So the installer actually launched Kite, but something isn't working. Setting priorities to maximum doesn't change anything.

I killed the processes and launched Kite again from cli and got this:

![kite_run](https://user-images.githubusercontent.com/42622067/81279665-b9ae8c00-9057-11ea-9023-520b127d23a4.png)

No other output and Kite's processes keep running in the fashion showed above and the registration window doesn't appear.

I'm talking about this window:

![image](https://user-images.githubusercontent.com/42622067/81279713-c9c66b80-9057-11ea-975e-0df0a65982c0.png)

This one never shows up to me.

What do you think?

**Comment by Amo95** [comment]
@davide-zagami I'm facing the same issue. I hope they fix this

**Comment by Amo95** [comment]
@davide-zagami you can install the suitable kite plugin for your preferred text editor by
- by cloning the kite repository for your editor.

~ I partially fixed mine by cloning the repository for sublime to my sublime package directory and installed the kite engine as well.

**Comment by davide-zagami** [comment]
That worked with my text editor, thanks! Too bad I still can't use Kite's Copilot, seems like a very useful tool to have.

**Comment by gennaro-ruccolo** [comment]
Yes, I have the same issue. Same OS and same Desktop Environment. Please fix it :-)

**Comment by Amo95** [comment]
it's fixed because it's working now 👍👨‍💻
